auth-lib/src/auth/java/com/spotify/sdk/android/auth/FallbackHandlerProvider.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2015-2016 Spotify AB
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.sdk.android.auth;
+
+import androidx.annotation.NonNull;
+import com.spotify.sdk.android.auth.browser.BrowserAuthHandler;
+
+/**
+ * Provides an AuthorizationHandler that opens a browser when the Spotify application is not installed
+ */
+public class FallbackHandlerProvider {
```

Comment:
Can't we have an interface under main and then implementations in auth and store? Or do these classes need to have the same names (I don't know how flavours work in this regard)? I was thinking having an API defined by interface might be nice.